src/analyze/analyze.c

```diff
@@ -651,14 +649,16 @@ static int analyze_plot(sd_bus *bus) {
 
         svg("<rect class=\"background\" width=\"100%%\" height=\"100%%\" />\n");
         svg("<text x=\"20\" y=\"50\">%s</text>", pretty_times);
-        svg("<text x=\"20\" y=\"30\">%s %s (%s %s %s) %s %s</text>",
-            isempty(host->os_pretty_name) ? "Linux" : host->os_pretty_name,
-            isempty(host->hostname) ? "" : host->hostname,
-            isempty(host->kernel_name) ? "" : host->kernel_name,
-            isempty(host->kernel_release) ? "" : host->kernel_release,
-            isempty(host->kernel_version) ? "" : host->kernel_version,
-            isempty(host->architecture) ? "" : host->architecture,
-            isempty(host->virtualization) ? "" : host->virtualization);
+        if (host != NULL) {
```

Comment:
Yes we, do get here. `host` isn't referenced anywhere between `acquire_host_info()` and `svg()` at line 654. Why bail? I don't want to bail before the very end of `analyze_plot()`, not beacuse of some data from `systemd-hostnamed` are missing.

We may not get here because other calls to `org.freedesktop.systemd1` fail but if we consider `acquire_host_info()` as a black box which leaves `host` unchanged (i.e. `NULL`) if it can't collect any information and, at the same time, we do not consider the failure `acquire_host_info()` fatal (which is what I want to achieve with this patch) then `host` must be checked otherwise dereferencing fields of `host` will trigger SIGSEGV.